clang/test/Driver/aarch64-v81a.c

```diff
@@ -19,3 +19,19 @@
 // RUN: %clang --target=arm64 -mlittle-endian -march=armv8.1a -### -c %s 2>&1 | FileCheck -check-prefix=ARM64-GENERICV81A %s
 // RUN: %clang --target=arm64 -mlittle-endian -march=armv8.1-a -### -c %s 2>&1 | FileCheck -check-prefix=ARM64-GENERICV81A %s
 // ARM64-GENERICV81A: "-cc1"{{.*}} "-triple" "arm64{{.*}}" "-target-cpu" "generic"{{.*}} "-target-feature" "+v8.1a"{{.*}} "-target-feature" "+neon"
+
+// ===== Architecture extensions =====
+
+// RUN: %clang -target aarch64 -march=armv8.1-a --print-enabled-extensions 2>&1 | FileCheck -check-prefix=ARCH-EXTENSION --implicit-check-not FEAT_ %s
+// ARCH-EXTENSION: FEAT_ETE
```

Comment:
I'm also curious what it looks like just to confirm that it would be possible to do something like `clang --print-enabled-extensions | tail | cut ... | sort | uniq` if you wanted to extract the list. I assume we can because it's in a table.

Reply:
Also, yes, I believe it should be possible to use something like `clang --print-enabled-extensions | tail | cut ... | sort | uniq`.

Reply:
> the implications between them are not very clear

True. The visual difference might be `+whatever` but the user doesn't know what that requires or implies.

Leave it as is and if anyone actually finds it useful they can add the `+...` themselves. I don't have an immediate use case myself.

Just being able to compare 1 compiler command line against a technical reference manual is already an improvement.